.github/workflows/test-course-definition.yml

```diff
@@ -55,6 +55,33 @@ jobs:
         run: test -z "$(git ls-files compiled_starters/ --exclude-standard --others)"
         working-directory: courses/${{github.event.repository.name}}
 
+  lint:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+
+    timeout-minutes: 5
+
+    steps:
+      - uses: oven-sh/setup-bun@v1
+
+      - uses: actions/checkout@v3
+        with:
+          repository: "codecrafters-io/course-sdk"
+          ref: ${{inputs.sdkRef}}
+
+      - uses: actions/checkout@v3
+        with:
+          path: courses/${{github.event.repository.name}}
+
+      - run: bun install
+      - run: make install
+
+      - name: "Lint using course-sdk"
+        run: course-sdk lint
```

Comment:
Shouldn't we also remove the other jobs in this file that are now replaced by `course-sdk` lint?

Reply:
https://github.com/codecrafters-io/course-sdk/pull/15#issuecomment-2146739618